### PR TITLE
refactor(traveller): use the tanstack table instead of the old one and add permissions

### PR DESCRIPTION
**Pull request by yusualhashash**
## How has this been tested?

- [ ] Prod
- [x] Development

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] Added all translations, no hard coded text. 
- [ ] Updated the UI (if needed).
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: apps/web/src/app/[lang]/app/[type]/parties/travellers/[travellerId]/identification/new/page.tsx [hunk 1]
```diff
@@ -22,22 +23,24 @@ export default async function Page({
...
         {`${languageData.Traveller} (${travellerData.personalIdentifications[0].fullName})`}
       </div>
       <div className="hidden" id="page-description">
         {languageData["Travellers.Create.Identification.Description"]}
       </div>
       <div className="hidden" id="page-back-link">
-        {getBaseLink(`/app/admin/parties/traveller/${params.travellerId}`)}
+        {getBaseLink(`/app/admin/parties/travellers/${params.travellerId}`)}
       </div>
     </>
```
Comment (eren-ayasofyazilim) [inline apps/web/src/app/[lang]/app/[type]/parties/travellers/[travellerId]/identification/new/page.tsx:45]: Page policy should be parent of everything. Put those divs inside of pagepolicy please.

File: apps/web/src/app/[lang]/app/[type]/parties/travellers/[travellerId]/identification/[identificationId]/page.tsx [hunk 2]
```diff
@@ -22,24 +23,26 @@ export default async function Page({
...
         {`${languageData["Travellers.Personal.Identification"]} (${travellerData.personalIdentifications[0].travelDocumentNumber})`}
       </div>
       <div className="hidden" id="page-description">
         {languageData["Travellers.Identifications.Edit.Description"]}
       </div>
       <div className="hidden" id="page-back-link">
-        {getBaseLink(`/app/admin/parties/traveller/${params.travellerId}`)}
+        {getBaseLink(`/app/admin/parties/travellers/${params.travellerId}`)}
       </div>
     </>
```
Comment (eren-ayasofyazilim) [inline apps/web/src/app/[lang]/app/[type]/parties/travellers/[travellerId]/identification/[identificationId]/page.tsx:47]: pagepolicy should be parent of everything

File: apps/web/src/app/[lang]/app/[type]/parties/travellers/[travellerId]/identification-table-data.tsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  if (
+    isActionGranted(["TravellerService.Travellers.Create"], grantedPolicies)
+  ) {
+    actions.push({
+      type: "simple",
+      actionLocation: "table",
+      cta: languageData["Travellers.New.Identification"],
+      icon: PlusCircle,
+      onClick() {
+        router.push(`${String(travellerId)}/identification/new`);
```
Comment (eren-ayasofyazilim) [inline apps/web/src/app/[lang]/app/[type]/parties/travellers/[travellerId]/identification-table-data.tsx:41]: why String(travellerId)

**Review by eren-ayasofyazilim — CHANGES_REQUESTED** [review]

File: apps/web/src/app/[lang]/app/[type]/parties/travellers/[travellerId]/identification-table-data.tsx (hunk 3 above)
Comment (yusualhashash) [reply apps/web/src/app/[lang]/app/[type]/parties/travellers/[travellerId]/identification-table-data.tsx:41]: the lint or github conver it i write it as here
""+travellerId+"/etc...

File: apps/web/src/app/[lang]/app/[type]/parties/travellers/[travellerId]/identification/[identificationId]/page.tsx (hunk 2 above)
Comment (yusualhashash) [reply apps/web/src/app/[lang]/app/[type]/parties/travellers/[travellerId]/identification/[identificationId]/page.tsx:47]: done

File: apps/web/src/app/[lang]/app/[type]/parties/travellers/[travellerId]/identification/new/page.tsx (hunk 1 above)
Comment (yusualhashash) [reply apps/web/src/app/[lang]/app/[type]/parties/travellers/[travellerId]/identification/new/page.tsx:45]: done

**Review by eren-ayasofyazilim — APPROVED** [review]

**Pull request merged**